docs/supporting-tools.md

```diff
@@ -0,0 +1,59 @@
+# Supporting tools and services
+
+This page outlines tools and services that currently support the development container specification, including the `devcontainer.json` format. A `devcontainer.json` file in your project tells tools and services that support the dev container spec how to access (or create) a dev container with a well-defined tool and runtime stack. 
+
+While most [dev container properties](devcontainerjson-reference.md) apply to any supporting tool or service, a few are specific to certain tools, which are outlined below.
+
+## devcontainer CLI
+
+Given the growing number of use cases for dev containers, there is a companion [devcontainer command line interface](https://code.visualstudio.com/docs/remote/devcontainer-cli) (CLI) that can be used independent of the Remote - Containers extension or GitHub Codespaces.
```

Comment:
This refers to the "old" CLI which we might need to rename to Remote-Containers CLI (for doing 'open'). Maybe replace with a note and link to the reference implementation issue?

Reply:
This makes sense. I updated wording in this section and linked to the issue where the implementation is being discussed: https://github.com/microsoft/dev-container-spec/issues/9. Let me know if this looks good to you.

I also added a `Remote - Containers CLI` section to mention and link to https://code.visualstudio.com/docs/remote/devcontainer-cli. Would love to hear if you think it makes sense to include this section, or we should just stick with the top `Dev container CLI` section.

Reply:
Yeah good point. Missed that. We need to think about how to describe the existing one as the reference implementation comes together - today it applies to multiple products as described here, but I agree that it will be important to give the reference implementation a proper name.